libs/blocks/marquee/marquee.css

```
@@ -235,6 +249,11 @@
   font-weight: 700;
 }
 
+.marquee.large.split p.action-area a:not(.con-button) {
```

I don't see a need to chain `action-area` to the p tag. Also, `con-buttons` are already getting explicitly styled. Adding the `not` isn't adding anything.
```suggestion
.marquee.large.split .action-area a {
```